doc/Tex/RAM_SCB_run.tex

```diff
@@ -1,95 +1,48 @@
 
 \begin{figure}[h]
-  \begin{center}
-    \includegraphics[width=25pc,clip=True,angle=90]{rundir.ps}
-  \end{center}
-  \caption{Run directory layout.  Gray and green rounded boxes indicate 
-  subdirectories; green boxes are directories that are soft linked one level
-  up for convenience.  Yellow boxes are key input files, the red box is the
-  linked RAM-SCB executable, and other data input files are listed without
-  boxes.}
-  \label{fig:rundir}
+ \begin{center}
+  \includegraphics[width=25pc,clip=True,angle=90]{rundir.ps}
+ \end{center}
+ \caption{Run directory layout. Gray and green rounded boxes indicate subdirectories; green boxes are directories that are soft linked one level up for convenience. Yellow boxes are key input files, the red box is the linked RAM-SCB executable, and other data input files are listed without boxes.}
+ \label{fig:rundir}
 \end{figure}
 
-Figure \ref{fig:rundir} shows the default run directory organization.  The
-IM directory is where most input and output occurs; subdirectories shaded
-green are soft-linked to the top run level for convenience.  These directories
-are self explanatory outside of {\tt output\_swmf}, which is for runs that use
-SWMF output in stand-alone mode.  Enough inputs are
-given in the input directories for a sample simulation right away.  
-The red box in Figure \ref{fig:rundir} denotes the linked executable.  The final
-files, denoted as yellow boxes, are the final input files.  {\tt w2k.dat} is
-the Weimer 2000 empirical model data file, {\tt omni.txt} is solar wind input 
-file from the OMNI database, and {\tt RamIndices.txt} is a list of 
-geomagnetic indices.  The necessity of these three files is discussed below.
-
-Finally, there is {\tt PARAM.in}, the run-time configuration file that is used
-to tell RAM-SCB what and how it should simulate.  The {\tt PARAM} interface,
-identical to the one used by the SWMF, is simple.  Any line of text in the
-file not preceeded by a \# symbol is ignored, so comment your file to your
-heart's content.    Any text preceeded by \#, however, is a interpreted as
-a {\tt PARAM} command.  A generic command takes the form,
+Figure \ref{fig:rundir} shows the default run directory organization. The IM directory is where most input and output occurs; subdirectories shaded green are soft-linked to the top run level for convenience. These directories are self explanatory outside of {\tt output\_swmf}, which is for runs that use SWMF output in stand-alone mode. Enough inputs are given in the input directories for a sample simulation right away. The red box in Figure \ref{fig:rundir} denotes the linked executable. The final files, denoted as yellow boxes, are the final input files. {\tt w2k.dat} is the Weimer 2000 empirical model data file, {\tt omni.txt} is solar wind input file from the OMNI database, and {\tt RamIndices.txt} is a list of geomagnetic indices. The necessity of these three files is discussed below.

```

Comment:
Are the required formats for, e.g., the OMNI file documented here? What is in which column, whether it requires a specific format statement, delimiter, etc.

Reply:
Yes, I think we need a new section for input formats. No information is currently available for that

Reply:
I'm going to leave this as future work so we can get a first cut committed. Marking as resolved.